test/unit_tests/autoqasm/test_api.py

```diff
@@ -1266,3 +1266,39 @@ def test(int[32] a, int[32] b) {
 test(2, 3);
 test(4, 5);"""
     assert main.build().to_ir() == expected
+
+
+class TestTypecasting:
+    def test_int_typecasting_on_measure(self):
+        @aq.main(num_qubits=2)
+        def main():
+            test = int(measure([0, 1]))
```

Comment:
to fix linter error:
```suggestion
            test = int(measure([0, 1]))  # noqa: F841
```